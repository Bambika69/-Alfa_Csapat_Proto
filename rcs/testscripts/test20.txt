createfield -t safehouse
createvirologist SA1
createvirologist SA1
createagent virus amnesia V1
createequipment cloak -v V2
setrandom False random
startturn V2
toggle E1
startturn V1
useagent A1 V2