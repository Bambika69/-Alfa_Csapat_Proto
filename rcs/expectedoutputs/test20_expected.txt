Field created:
ID: F1
Type: Field
Neighbours: null

Virologist created:
ID: V1
Field: F1
Backpack: B1

Virologist created:
ID: V2
Field: F1
Backpack: B2

Agent created:
ID: AV1
Type: virus
GeneticCode: amnesia
Virologist: V1

Equipment created:
ID: C1 
Type: Cloak
Destination: V2
Result: successful

Random behavior changed:
yesOrNoType: False
choiceType: random

V2’s turn started.

Equipment toggled on Virologist:
Equipment: C1
Virologist: V2
Inf: wear

V1’s turn started.

Agent used on Virologist:
Agent: AV1
Virologist: V1
Target: V2
Result: successful
